docs/en/user-manual/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: User Manual
+advertise: User Manual
+lang: en
+nav_link: User Manual
+nav_level: 1
+nav_order: 10
+---
+
+Version 2.1rc3, released 2017-05-22
```

Comment:
Sorry to be a best, but since you didn't take my advice to move all the versioning info to the About page, you need to update this instance as well to:

Version 2.1.1, released 2017-06-09

which is my 61st birthday, by the way! :)